components/Header.tsx

```diff
@@ -24,9 +26,11 @@ function Header({
           <a className="flex items-center">
             <img className="h-10 w-auto sm:h-12 my-2" src="/logo.svg" alt="" />
             <div className="ml-5 flex flex-col justify-center">
-              <div className="font-bold text-gray-900 leading-tight text-2xl sm:text-3xl tracking-tight">
-                Deno
-              </div>
+              {!main && (
+                <div className="font-bold text-gray-900 leading-tight text-2xl sm:text-3xl tracking-tight">
+                  Deno
+                </div>
+              )}
```

Comment:
```suggestion
```

Can we remove it instead of using a flag?

Reply:
I want it to say Deno on pages other than index.